R/script_template.R

```diff
@@ -1,9 +1,14 @@
-script_template <- function(author = Sys.info()[["user"]]) {
+script_template <- function(author = ifelse(!is.null(git2r::config()$global$user.name),
+                              git2r::config()$global$user.name, Sys.info()[["user"]]
+                            )) {
   author <- paste("# Original author(s):", author)
   orig_date <- paste("# Original date:", Sys.Date())
-  run_on <- paste0("# Written/run on RStudio ", RStudio.Version()$mode, " ",
-                   RStudio.Version()$version, " and R ",
-                   version$major, ".", version$minor)
+  run_on <- paste0(
+    "# Written/run on RStudio ",
+    rstudioapi::versionInfo()$mode, " ",
+    rstudioapi::versionInfo()$version, " and R ",
+    version$major, ".", version$minor
+  )
```

Comment:
This could / should probably be its own function as it's used in a number of places now.